instrumentation/mongodb/pom.xml

```
@@ -31,16 +31,23 @@
 
     <main.basedir>${project.basedir}/../..</main.basedir>
 
-    <mongodb-driver.version>3.12.14</mongodb-driver.version>
+    <mongodb-driver.version>5.0.1</mongodb-driver.version>
+    <old-mongodb-driver.version>3.11.0</old-mongodb-driver.version>
```

Nit, may be `mongodb3-driver.version` instead of `old`?

```suggestion
    <mongodb3-driver.version>3.11.0</mongodb3-driver.version>
```

---

I was using old- elsewhere, like old-okhttp old-amqp to make it obvious this isn't a prod dep ;)

version suffix is implicitly "old" like no longer maintained except in the case of spring, where two are maintained.. maybe I can update everything to a different word? anyway I don't want to be distracting, but that was the rationale. I can switch this, just there are other places where old is used.

maybe s/old/earliest or floor?

---

All good, just a nit :D

---

I think I like floor version better, so will go with that. because we don't want to have to rename everything each time the floor drops out.